### PR TITLE
Sk dockerize

### PR DESCRIPTION
**Pull request by JuneKelly**
Child of https://github.com/overleaf/sharelatex/issues/994

- Add docker/docker-compose config
- Add a simple makefile to help with running git-bridge

Child of https://github.com/overleaf/sharelatex/issues/1175

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,3 +93,40 @@ The configuration file is in `.json` format.
     }
 
 You have to restart the server for configuration changes to take effect.
+
+
+## Setup with v2 Dev-Environment
```
Comment (jpallen) [inline README.md]: With the changes in the dev-env, I think this is all done automatically now, and so doesn't need including here? Likewise with the OAuth app - the instructions for that should be to run `bin/run rails_v1 rake db:seed` in the dev environment?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: "3"
```
Comment (jpallen) [inline docker-compose.yml]: I don't think we need this docker-compose file now, since we'll run it from the top-level dev env?

**Review by jpallen — CHANGES_REQUESTED** [review]
LGTM, but I think we have some redundant things we can remove now this is configured via the dev env?

**Comment by JuneKelly** [comment]
@jpallen up for re-review :+1:

**Review by jpallen — APPROVED** [review]

**Pull request merged**